src/metabase/upload.clj

```diff
@@ -210,11 +218,32 @@
 
 (def ^:private separators ",;\t")
 
+(def ^:private max-infered-lines 10)
```

Comment:
maybe leave a comment on why 10 was chosen, and if there's no good reason, maybe leave a comment saying so. Our future selves might have the confidence to increase it